R/archive.R

```diff
@@ -21,7 +21,7 @@ NULL
 #' a <- archive(system.file(package = "archive", "extdata", "data.zip"))
 #' a
 #' @export
-archive <- function(file, options = character()) {
+archive <- function(file, options = character(), password = "") {
```

Comment:
I wonder if it is more natural to set `password = NA_character_` by default for no password. (Then I guess the empty string would be an error.)

Reply:
@gaborcsardi I changed to `password = NA_character_` but now all features are broken because `cpp11::is_na(password)` gets false even though I pass NA_character_. Do you have ideas? I really have no knowledge on R binding.

Reply:
Some debugging showed that r_string being argument does not work well on protection, so I changed to convert to vector in R phase.